src/anm/render.js

```diff
@@ -67,14 +67,15 @@ function r_loop(ctx, player, anim, before, after, before_render, after_render) {
 
     return nextFrame(function() {
         r_loop(ctx, player, anim, before, after, before_render, after_render);
-    })
+    });
 }
+
 function r_at(time, dt, ctx, anim, width, height, zoom, rib_color, before, after) {
     ctx.save();
     var ratio = engine.PX_RATIO;
     if (ratio !== 1) ctx.scale(ratio, ratio);
-    var width = width | 0,
-        height = height | 0;
+    width = width | 0;
+    height = height | 0;
```

Comment:
Note to myself: some comment that this is a rounding and not defaulting here should be added.